Update dlp-conditions-and-exceptions.md

aamalhot opened the pull request:
Add details for sender address location

aamalhot committed:
Update dlp-conditions-and-exceptions.md

Add details for sender address location

aamalhot commented:
Please hold for approval

aamalhot committed:
Update dlp-conditions-and-exceptions.md

update image

aamalhot commented:
@chrfox please review and merge

yogkumgit commented:
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

JohanFreelancer9 commented on microsoft-365/compliance/dlp-conditions-and-exceptions.md:
```diff
@@ -43,8 +43,28 @@ The tables in the following sections describe the conditions and exceptions that
 
 ### Senders
 
+For conditions and exceptions that examine the sender's address, you can specify where rule looks for the sender's address. The sender address used by Data Loss Prevention (DLP) to evaluate sender based conditions listed below is the Envelope address. Note that Exchange transport rules (ETR) uses the header address as the default sender address location.
```
```suggestion
For conditions and exceptions that examine the sender's address, you can specify where the rule looks for the sender's address. The sender address, used by the DLP rule to evaluate the sender-based conditions listed below, is the Envelope address. Note that Exchange transport rules use the Header address as the default sender address location.
```

JohanFreelancer9 commented on microsoft-365/compliance/dlp-conditions-and-exceptions.md:
```diff
@@ -43,8 +43,28 @@ The tables in the following sections describe the conditions and exceptions that
+For conditions and exceptions that examine the sender's address, you can specify where rule looks for the sender's address. The sender address used by Data Loss Prevention (DLP) to evaluate sender based conditions listed below is the Envelope address. Note that Exchange transport rules (ETR) uses the header address as the default sender address location.
 <br>
 
+>[!Note]
+>Starting 20th January 2021, the default sender address location will be moved to the Header address along with the availability of the -SenderAddressLocation parameter to configure desired behavior at a DLP rule level.
```
```suggestion
> [!NOTE]
> Starting January 20, 2021, the default sender address location will be moved to the Header address along with the availability of the -SenderAddressLocation parameter to configure desired behavior at a DLP rule level.
```

JohanFreelancer9 commented on microsoft-365/compliance/dlp-conditions-and-exceptions.md:
```diff
@@ -43,8 +43,28 @@ The tables in the following sections describe the conditions and exceptions that
+
+![image](https://user-images.githubusercontent.com/53205984/145942298-6b435ba6-d146-44fe-a1c5-58babeaf8d7a.png)
+
+At the tenant level, you can configure a Sender Address Location to be used across all rules, unless overridden by a single rule. To revert tenant DLP policy configuration to evaluate sender address from Envelope across all rules, you can run the below command-
```
```suggestion
At the tenant level, you can configure a sender address location to be used across all rules, unless overridden by a single rule. To revert tenant DLP policy configuration to evaluate the sender address from the Envelope across all rules, you can run the following command:
```

JohanFreelancer9 commented on microsoft-365/compliance/dlp-conditions-and-exceptions.md:
````diff
@@ -43,8 +43,28 @@ The tables in the following sections describe the conditions and exceptions that
+Set-PolicyConfig –SenderAddressLocation Envelope
+```
+
+To configure sender address location at a DLP rule level, the parameter is _SenderAddressLocation_. The available values are:
````
```suggestion
To configure the sender address location at a DLP rule level, the parameter is _SenderAddressLocation_. The available values are:
```

JohanFreelancer9 reviewed (changes requested):
Copy review.

yogkumgit commented:
@aamalhot Please implement the suggested changes. Thanks

aamalhot committed:
Update microsoft-365/compliance/dlp-conditions-and-exceptions.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

yogkumgit commented:
@JohanFreelancer9

JohanFreelancer9 commented:
@aamalhot, kindly commit all my suggested changes. You only seem to have committed one suggestion. Many thanks.

aamalhot committed:
Update microsoft-365/compliance/dlp-conditions-and-exceptions.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

aamalhot committed:
Update microsoft-365/compliance/dlp-conditions-and-exceptions.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

aamalhot committed:
Update microsoft-365/compliance/dlp-conditions-and-exceptions.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

aamalhot commented:
Thanks @JohanFreelancer9. Completed all

aamalhot committed:
Merge branch 'public' into patch-17

yogkumgit commented:
@JohanFreelancer9

JohanFreelancer9 commented on microsoft-365/compliance/dlp-conditions-and-exceptions.md:
```diff
@@ -43,8 +43,28 @@ The tables in the following sections describe the conditions and exceptions that
+
+- **Header or envelope** (`HeaderOrEnvelope`) Examine senders in the message header and the message envelope.
+<br>
 ****
```
```suggestion
```

aamalhot committed:
Update microsoft-365/compliance/dlp-conditions-and-exceptions.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

aamalhot commented:
@JohanFreelancer9 updated

JohanFreelancer9 reviewed (approved):
Thank you, @aamalhot.

yogkumgit commented:
@chrfox Please merge this PR. Thanks

aamalhot committed:
Update dlp-conditions-and-exceptions.md

aamalhot committed:
Merge branch 'public' into patch-17

chrfox committed:
Update dlp-conditions-and-exceptions.md

@aamalhot  I changed the opening para and commented out the [!NOTE] through message envelope. <br> because they are date dependent.  The comment out characters can just be removed on 1/20 or when this goes live.  Please review these changes and sign off and I'll publish

Chris

aamalhot commented:
Looks good @chrfox . Please proceed with publish

chrfox committed:
Merge branch 'public' into patch-17
